tools/ci_build/github/azure-pipelines/linux-cpu-aten-pipeline.yml

```diff
@@ -28,12 +32,40 @@ jobs:
     inputs:
       versionSpec: '16.x'
 
+  - task: Cache@2
+    displayName: Cache Docker Image Task
+    inputs:
+      key: '$(TODAY) | "$(Build.SourceBranch)" | "$(Build.SourceVersion)"'
+      path: $(Pipeline.Workspace)/docker
```

Comment:
Suggest changing $(Pipeline.Workspace) to $(Agent.TempDirectory) as this tar is temporary.

Reply:
Changed